sympy/physics/continuum_mechanics/beam.py

```diff
@@ -396,6 +396,11 @@ def apply_load(self, value, start, order, end=None):
         start = sympify(start)
         order = sympify(order)
 
+        if start.is_real and start > self.length or (end != None and end > self.length):
```

Comment:
Do you have to check to see if end is real?

Reply:
Yes definitely I'll update it.